static/js/util/util_test.js

```diff
@@ -82,6 +85,44 @@ describe("util", () => {
     })
   })
 
+  describe("getPaymentWithFulfilledOrder", () => {
+    let klass, payment
+
+    beforeEach(() => {
+      klass = generateFakeKlasses(1, { hasPayment: true })[0]
+      payment = klass.payments[0]
+    })
+
+    it("gets the payment for the fulfilled order", () => {
+      assert.deepEqual(
```

Comment:
Would be better to be explicit about setting the status to "fulfilled" rather than relying on the generate function to do that automatically.

Reply:
Will do